vnext/ReactUWP/Utils/PropertyUtils.h

```diff
@@ -42,6 +42,30 @@ static const std::unordered_map<std::string, ShadowEdges> edgeTypeMap = {
     {"borderWidth", ShadowEdges::AllEdges},
 };
 
+static const std::unordered_map<std::string, ShadowCorners> borderRadiusTypeMap = {
+    {"borderTopLeftRadius", ShadowCorners::TopLeft},
+    {"borderTopRightRadius", ShadowCorners::TopRight},
+    {"borderBottomLeftRadius", ShadowCorners::BottomLeft},
+    {"borderBottomRightRadius", ShadowCorners::BottomRight},
+    {"borderTopStartRadius", ShadowCorners::TopStart},
+    {"borderTopEndRadius", ShadowCorners::TopEnd},
+    {"borderBottomStartRadius", ShadowCorners::BottomStart},
+    {"borderBottomEndRadius", ShadowCorners::BottomEnd},
+    {"borderRadius", ShadowCorners::AllCorners},
+};
+
+static const std::unordered_map<std::string, ShadowEdges> paddingTypeMap = {
+    {"paddingLeft", ShadowEdges::Left},
+    {"paddingTop", ShadowEdges::Top},
+    {"paddingRight", ShadowEdges::Right},
+    {"paddingBottom", ShadowEdges::Bottom},
+    {"paddingStart", ShadowEdges::Start},
+    {"paddingEnd", ShadowEdges::End},
+    {"paddingHorizontal", ShadowEdges::Horizontal},
+    {"paddingVertical", ShadowEdges::Vertical},
+    {"padding", ShadowEdges::AllEdges},
+};
+
```

Comment:
I'd like us to rewrite all these (not necessary in this PR but since you're cleaning this up I'm calling it out :) ). These each generate dynamic initializers and atexit destructors, and unnecessarily increase DLL load time and add unnecessary bloat. An array of structs of PCWSTR+enum would work fine in all these cases. Global std::maps, esp those of std::strings, should be removed everywhere.